packages/cli/src/util/git-helpers.ts

```diff
@@ -0,0 +1,87 @@
+import { execSync } from 'node:child_process';
+
+/** Defines the options for executing Git commands */
+export type GitExecOptions = Readonly<{
+  /** If set to true, the function will throw
+   * an error if any occurs during the execution of the Git command. By default,
+   * it is set to false, meaning errors are caught and handled gracefully.*/
+  unsafe?: boolean;
+  /** Specifies the current working directory
+   * from which the Git command should be executed. If not specified, it
+   * defaults to the current working directory of the process.*/
+  cwd?: string;
+}>;
+
+const DEFAULT_GIT_EXEC_OPTS = {
+  unsafe: false,
+  cwd: process.cwd(),
+} as const;
```

Comment:
Was there a reason for `as const` here? Doesn't seem necessary to me at first glance

```suggestion
};
```

Reply:
Habit haha